### ISSUE TITLE
Canal Sur (por lo menos A la Carta) no funciona

### ISSUE THREAD

**Issue by aabilio** [aabilio/PyDownTV2#42]
[Visto en un comentario en la extensión para Chrome]
Vídeo de Canal Sur, no están funcionando.

**Comment by ghost** [comment]
Sí, estoy intentándolo con uno desde el ejecutable de Windows y me da este error...

![pydowntv-error](https://f.cloud.github.com/assets/407360/817635/8b0286c0-ef7a-11e2-86b0-11e9c2acc4ca.png)
